docker-compose.yml

```diff
@@ -47,6 +47,22 @@ bionic:
     - mongodb
     - postgres
 
+el8:
+  image: packagingrunner:trusty
```

Comment:
Please use `quay.io/stackstorm/packagingrunner` as in other runners as well.

Reply:
ok. Need images up on quay

Reply:
Any specific reason about why original `quay.io/stackstorm/packagingrunner` didn't work?